mcs/class/corlib/System/Environment.cs

```diff
@@ -553,11 +553,7 @@ internal static string InternalGetFolderPath (SpecialFolder folder)
 #endif
 			// personal == ~
 			case SpecialFolder.Personal:
-#if MONOTOUCH
```

Comment:
I believe that is similar to what we do in other places where we don't want MONOTOUCH specific behavior.